### PR TITLE
Pull request for synthesizing assignment WK2

### PR DESCRIPTION
**Pull request by adrianjengel**
Please review.

### CODE REVIEW

File: task_01.rst [hunk 1]
```diff
@@ -0,0 +1,104 @@
+===================
```
Comment (cheuschober) [inline task_01.rst]: Make sure your heading decoration (eg, `=`) are aligned with the length of the heading inside. Not all interpreters are so forgiving.

**Comment by cheuschober** [comment]
retest this please

File: task_01.rst (hunk 1 above)
Comment (adrianjengel) [inline task_01.rst]: Does it have to be the same amount of characters or can you just eye-ball it?

Comment (cheuschober) [inline task_01.rst]: It should be the same amount of characters. I didn't dock you for it but felt you should just know, per the standard.

Comment (adrianjengel) [inline task_01.rst]: Got it. Thank you.

File: task_01.rst [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+##############
+
+  1. Python can be used for almost any task you can think of.
+  2. It has a huge library of available modules.
+  3. Pip extends that library even more.
+  4. Why aren't you using it already! Go write some Python!     
+
+.. table:: Sources
+
+  +---------+-------------------+
```
Comment (cheuschober) [inline task_01.rst]: Another point to note here, keep your indentation levels consistent. For example, you indented your table 2 spaces but your note 4 spaces.  You also don't have to indent lists (bulleted or numbered) as you did at line 83, they'll naturally indent themselves. :)

File: task_03.py [hunk 3]
```diff
@@ -1,7 +1,8 @@
-CHARLIE = "Chocolate Factory" 
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Hi I am a docstring."""
+
+CHARLIE = "Brown"
```
Comment (cheuschober) [inline task_03.py]: Excellent catch here! You did well interpreting the error code.

**Pull request merged**